### ISSUE TITLE
Choose the limit to show in X-RateLimit-* headers

### ISSUE THREAD

**Issue by ncrocfer** [alisaifee/flask-limiter#105]
Hey ! First of all thank you for this awesome extension :)

I have a need but I'm not sure how to implement it. Can you say me if it's possible with Flask-Limiter please ?

I want to set a default rate limit, say **1 per second**. But when I enable the ratelimit headers, the response is something like :

```
HTTP/1.0 200 OK
Content-Type: application/json
...
X-RateLimit-Limit: 1
X-RateLimit-Remaining: 0
X-RateLimit-Reset: 1509823497
```

In that case the headers are not very useful. I would like something more readable, for example :

```
HTTP/1.0 200 OK
Content-Type: application/json
...
X-RateLimit-Limit: 3600
X-RateLimit-Remaining: 3599
X-RateLimit-Reset: 1509829396
```

So an idea is to change the rate limit to 3600/hours, the headers will be more explicit, but I can receive a lot's of requests in a very short period by a DOS attack.

Another idea is to mix the limits, for example **1/second;3600hour**, but like the documentation says it :

> Since multiple rate limits can be active for a given route - the rate limit with the lowest time granularity will be used in the scenario when the request does not breach any rate limits.

So even if I mix the rate limits, the headers will be based on **1/second**. Do you think if it's possible to display the **3600/hour** related headers, but while keeping the **1/second** to avoid a DOS please ?

**Comment by Asoul** [comment]
Maybe can set the display order for the limiter, or a default value.

**Comment by alisaifee** [comment]
would a `DISPLAY_ORDER={ASC, DESC}` configuration option work for you?

**Comment by lamergameryt** [comment]
> would a `DISPLAY_ORDER={ASC, DESC}` configuration option work for you?

Hello, I'm curious to know if this has been implemented yet. If so, how do you change the display order?

**Comment by alisaifee** [comment]
Instead of adding more complexity to the default header configuration, you can instead use the `current_limit` or `current_limits` attribute of the extension in your own `after_request` hook to setup the headers the way you prefer. Note that this is available only in `2.1.x`. References:
 - [current_limit](https://flask-limiter.readthedocs.io/en/latest/api.html#flask_limiter.Limiter.current_limit)
 - [current_limits](https://flask-limiter.readthedocs.io/en/latest/api.html#flask_limiter.Limiter.current_limits)
